wallet/src/Pos/Wallet/Web/Pending/Util.hs

```diff
@@ -32,9 +33,16 @@ import           Pos.Wallet.Web.Pending.Updates (mkPtxSubmitTiming)
 import           Pos.Wallet.Web.State           (getWalletMeta)
 
 ptxPoolInfo :: PtxCondition -> Maybe PtxPoolInfo
-ptxPoolInfo (PtxApplying i)    = Just i
-ptxPoolInfo (PtxWontApply _ i) = Just i
-ptxPoolInfo _                  = Nothing
+ptxPoolInfo (PtxCreating i)     = Just i
+ptxPoolInfo (PtxApplying i)     = Just i
+ptxPoolInfo (PtxWontApply _ i)  = Just i
+ptxPoolInfo PtxInNewestBlocks{} = Nothing
+ptxPoolInfo PtxPersisted{}      = Nothing
+
+-- | Whether transaction is claimed to be once created.
+isPtxActive :: PtxCondition -> Bool
+isPtxActive PtxCreating{} = False
+isPtxActive _             = True
```

Comment:
Can a transaction be considered `active` is if in `Applying` as well? What `active` really means in this context? I guess it really means "everything but `Creating`"?

Reply:
> Whether transaction is claimed to be once created.

Yep.